package.json

```diff
@@ -5,16 +5,18 @@
   "main": "ping-centre.js",
   "license": "MPL-2.0",
   "scripts": {
-    "test": "npm-run-all test:*",
-    "test:code": "make test",
-    "test:lint": "eslint ."
+    "bundle": "webpack",
+    "test": "npm run bundle && npm run test:node && npm run test:browser && npm run test:lint",
+    "test:node": "istanbul cover _mocha -- --recursive",
+    "test:browser": "karma start test/karma.conf.js",
+    "travis": "npm test && cat ./coverage/lcov.info | coveralls",
+    "test:lint": "eslint . --ext .js"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/mozilla/ping-centre.git"
   },
   "author": "Marina Samuel <msamuel@mozilla.com>",
-  "license": "MPL-2.0",
```

Comment:
@6a68 I've never seen a license disappear before (although npm has been getting better at validating SPDX licenses). I think this line was removed because it was already defined up on Line 6 above. #yolo